modules/imgproc/src/opencl/box_filter.cl

```diff
@@ -0,0 +1,78 @@
+// This file is part of OpenCV project.
```

Comment:
Could you change:
- file name to `boxFilter_8UC1.cl` (2 files with similar names (box_filter.cl / boxFilter.cl) looks wrong)
- kernel name to `boxFilter_8UC1_cols16_rows2`